### ISSUE TITLE
Transform @apply's from a node_modules package

### ISSUE THREAD

**Issue by MichaelGitArt** [windicss/nuxt-windicss#185]
Reading the docs, I found this:
https://windicss.org/integrations/nuxt.html#transform-apply-s-from-a-node-modules-package

And there is a usage of "fg" on the page of nuxt3 integration. The question is, what is it? How to import this function?
I don't see anything similar in the nuxt-windicss docs.

**Comment by harlan-zw** [comment]
Hey @MichaelGitArt 

Did you close this as you figured out a solution? Happy to help if you're still stuck :)

**Comment by MichaelGitArt** [comment]
Thanks, @harlan-zw  . I just realized that it is not related not the nuxt-windicss. I created similar ticket in the windicss core repository.

And in the end, I tried to implement it by myself and realized that extraTransformTargets.detect doesn't work with .vue correctly (not compiling classes' styles. So, I decided to do it by creating a symlink to the node_modules repository in the root.
![image](https://user-images.githubusercontent.com/34610289/194805613-bf8ff57d-9855-4d27-b216-e3ea14a73850.png)
